yt_dlp/extractor/skeb.py

```diff
@@ -0,0 +1,63 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import ExtractorError, determine_ext, ext2mimetype, traverse_obj
+from ..compat import compat_str
+
+
+class SkebIE(InfoExtractor):
+    _VALID_URL = r'https?://skeb\.jp/@[^/]+/works/(?P<id>\d+)'
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+        nuxt_data = self._search_nuxt_data(webpage, video_id)
```

Comment:
```suggestion
        nuxt_data = self._search_nuxt_data(self._download_webpage(url, video_id), video_id)
```
webpage is never used anywhere else?

Reply:
yes